python/hsfs/core/arrow_flight_client.py

```diff
@@ -54,6 +54,7 @@ class ArrowFlightClient:
     FILTER_NUMERIC_TYPES = ["bigint", "tinyint", "smallint", "int", "float", "double"]
     READ_ERROR = 'Could not read data using ArrowFlight. If the issue persists, use read_options={"use_hive": True} instead.'
     WRITE_ERROR = 'Could not write data using ArrowFlight. If the issue persists, use write_options={"use_spark": True} instead.'
+    DEFAULT_TIMEOUT = 900
```

Comment:
> The server has a queue timeout of 10 minutes by default and the client should match that configuration.
should this be 600?

Reply:
No. I added 300 seconds to the 600 seconds maximum queue time that the server has configured as it might hang for 600 seconds in the worst case and should then have some head room to execute.